File: TwoCaptcha/Captcha/Yandex.cs
namespace TwoCaptcha.Captcha
{
    public class Yandex : Captcha
    {
        public Yandex() : base()
        {
            parameters["method"] = "yandex";
        }

        public void SetSiteKey(string siteKey)
        {
            parameters["sitekey"] = siteKey;
        }

        public void SetUrl(string url)
        {
            parameters["pageurl"] = url;
        }
    }
}

File: TwoCaptcha/Captcha/Yandex.java
package com.twocaptcha.captcha;

public class Yandex extends Captcha {

    public Yandex() {
        super();
        params.put("method", "yandex");
    }

    public void setSiteKey(String siteKey) {
        params.put("sitekey", siteKey);
    }

    public void setUrl(String url) {
        params.put("pageurl", url);
    }

}
